examples/Make_Grid.ipynb

```diff
@@ -0,0 +1,444 @@
+    "az = 0. * u.deg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next cell will convert the specified Alt/Az into RA/Dec coordinates in ICRS. This coordinate will be used to slew and track. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.sleep(1.)\n",
+    "time_data = await atptg.tel_timeAndDate.next(flush=True, timeout=2)\n",
+    "curr_time_atptg = Time(time_data.tai, format=\"mjd\", scale=\"tai\")\n",
+    "time_err = curr_time_atptg - Time.now()\n",
+    "print(f\"Time error={time_err.sec:0.2f} sec\")\n",
+    "\n",
+    "# Compute RA/Dec for commanded az/el\n",
+    "cmd_elaz = AltAz(alt=alt, az=az, \n",
+    "                 obstime=curr_time_atptg.tai, \n",
+    "                 location=location)\n",
+    "cmd_radec = cmd_elaz.transform_to(ICRS)\n",
+    "\n",
+    "await atptg.cmd_raDecTarget.set_start(\n",
+    "    targetName=\"local\",\n",
+    "    targetInstance=ATPtg.TargetInstances.CURRENT,\n",
+    "    frame=ATPtg.CoordFrame.ICRS,\n",
+    "    epoch=2000,  # should be ignored: no parallax or proper motion\n",
+    "    equinox=2000,  # should be ignored for ICRS\n",
+    "    ra=cmd_radec.ra.hour,\n",
+    "    declination=cmd_radec.dec.deg,\n",
+    "    parallax=0,\n",
+    "    pmRA=0,\n",
+    "    pmDec=0,\n",
+    "    rv=0,\n",
+    "    dRA=0,\n",
+    "    dDec=0,\n",
+    "    rotPA=180.-alt.value,\n",
+    "    rotFrame=ATPtg.RotFrame.FIXED,\n",
+    "    rotMode=ATPtg.RotMode.FIELD,\n",
+    "    timeout=10\n",
+    ")\n",
+    "\n",
+    "print(f\"raDecTarget ra={cmd_radec.ra!r} hour; \"\n",
+    "      f\"declination={cmd_radec.dec!r} deg\")\n",
+    "# script.ataos.cmd_enableCorrection.set(hexapod=True)\n",
+    "# await script.ataos.cmd_enableCorrection.start(timeout=10)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.gather(atmcs.close(), \n",
+    "                    atptg.close())\n",
+    "\n",
+    "await d.close()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await atptg.cmd_stopTracking.start(timeout=10.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await atptg.cmd_pointLoadModel.set_start(pointingFile='/home/saluser/repos/ts_pointing_common/install/data/at.mod')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await atptg.cmd_pointLoadModel.set_start(pointingFile='auxtel_20190911.dat')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.sleep(5.)\n",
+    "\n",
+    "for i in range(10):\n",
```

Comment:
I must be really daft but I don't see how this is changing targets. I don't see any offset that is iterated over...